components/presence/presence_default.story.vue

```diff
@@ -0,0 +1,14 @@
+<template>
+  <dt-presence
+    presence="active"
```

Comment:
the default story needs to be modifiable by storybook controls which means you have to reference the built in properties and have to include every prop/slot

```
:presence="presence"
:theme="theme"
:srText="srText"
```

Reply:
Added